docs/openCypher.md

```diff
@@ -0,0 +1,54 @@
+# openCypher Data Format
+
+openCypher was chosen as it represents “the most widely adopted, fully-specified, and open query
+language for property graph databases”. Allowing customers to import data from enterprise applications
+such as neo4j and Amazon Neptune with greater ease. One caveat to this is that there are slight differences
+in the System column header formats used by each, these differences are shown below.
+#### Entity (Node) System column headers
+- :ID / _id  –   (Required) An ID for the node.
+- :LABEL / _labels  –   A label for the node.
+
+#### Edge (relationship) System column headers
+- :ID / _id   –   An ID for the relationship.
+- :START_ID / _start  –   (Required) The node ID of the node this relationship starts from.
+- :END_ID / _end   –   (Required) The node ID of the node this relationship ends at.
+- :TYPE / _type   –   A type for the relationship.
+
+#### Property column headers
+Both Entities and Edges can have associated properties. It's possible to specify the type of
+each property provided by using the proceeding format **propertyname:type**, the default is to
+treat each property as a string if a type isn't supplied.
+
+#### Supported Data Types
```

Comment:
Maybe rather than repeating what the Neptune page says, we could say which Gaffer types these are serialised with?